kore/src/Kore/Step/SMT/Evaluator.hs

```diff
@@ -132,6 +132,10 @@ decidePredicate
     -> simplifier (Maybe Bool)
 decidePredicate korePredicate =
     SMT.withSolver $ runMaybeT $ do
+        (logInfo . Text.pack . show . Pretty.vsep)
```

Comment:
Should we add a new data type and some logging functions similar to the ones in `Kore.Logger.(Debug.*|Warn.*)`, perhaps in `DebugSolver`?